trainings/sourcegraph-101/talk-track.md

```diff
@@ -0,0 +1,327 @@
+# Sourcegraph 101
+
+**Topic:** This training is focused on end users, and serves as a start-from-zero training for new end users. Admin tasks and more advanced search functionality are covered in other trainings. The training is broken into 5 units, but you don't need to call the units out to the customer—they serve more as a way to structure the training for yourself.
```

Comment:
There is a separate version of this content that is formatted for an LMS. This is intended as a talk track, and so I think it's worth formatting it as if that's what it will remain, including notes for CEs. 

@amenne , thoughts?

Reply:
LMS styled content wouldn't be scripts or lesson plans, right? Well maybe a lesson plan but complementary content for it. It seems like this repo and content is for CEs (or in the future trainers) to use / reference for live training.

As for an earlier comment:
> We need to decide if we want to keep a script format or a lesson plan format where the script is more determined by the trainer and the objectives, information, etc is here.

I think both are good: scripts for newer folks to have as guides to make their own and lesson plans for more tenured people to use as a guide.. Thoughts?

Reply:
I don't mind a script if it's helpful for people less tenured, but would prefer more of a reference of talking points with some information that's self-serviceable. I think for now keeping the script style works for shipping something and we can iterate on it by expanding the script to talking points in a "lesson plan" style and do that for other trainings going forward.

Reply:
@amenne Correct, it's intended to be consumed as written content. (And yes, supplemental to the lesson plan.) This specific content is intended as a reference point for CEs, yes. (You've seen the LMS content, which is in GDrive for now but will be moved over here as well for future reference.) 

I'm going to ship this as-is and can build a lesson plan version alongside, if there's a feeling that lesson plan content is valuable.